packages/components/pages/f-checkout/src/services/addressService/tenants/ie.js

```diff
@@ -0,0 +1,42 @@
+function isFullPostCode (postcode) {
+    return postcode?.length >= 5;
+}
+
+function toFormattedPostcode (postcode) {
+    let formattedPostcode = postcode.toString().replace(/\s/g, '').replace(/-/g, '');
```

Comment:
Should the matching regex be `\s+`?

Reply:
These are copied from existing functions. Both comments.